src/components/InputTodo.js

```diff
@@ -0,0 +1,3 @@
+const InputTodo = () => <div>input field here...</div>;
+
+export default InputTodo;
```

Comment:
- [ ] I suggest that you implement the `InputTodo` component so that it is not left empty.With this you will show your skills in using `props`,`state`,`events`,`hooks`.